infinispan-remote/pom.xml

```diff
@@ -168,10 +145,88 @@
             <artifactId>byteman-install</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.infinispan.server</groupId>
+            <artifactId>infinispan-server-build</artifactId>
+            <version>${infinispanVersion}</version>
+            <type>zip</type>
+            <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>*</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
I think it's an error of the Maven pom of that artifact.. it attempts to transitively bring in all its dependencies. Including the Hibernate Search version we needed to get rid of..